src/SDKs/ManagedServices/Management.ManagedServices/Microsoft.Azure.Management.ManagedServices.csproj

```diff
@@ -1,22 +1,26 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-	<!-- Please do not move/edit code below this line -->
-	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
-	<!-- Please do not move/edit code below this line -->
+<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code below this line -->
 
-	<PropertyGroup>
-		<PackageId>Microsoft.Azure.Management.ManagedServices</PackageId>
-		<Description>Microsoft Azure Management ManagedServices library</Description>
-		<AssemblyName>Microsoft.Azure.Management.ManagedServices</AssemblyName>
-		<Version>0.9.0-preview</Version>
-		<PackageTags>Azure ManagedServices;</PackageTags>
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.ManagedServices</PackageId>
+    <Description>Microsoft Azure Managed Services Management Library</Description>
+    <Version>0.9.0</Version>
+    <AssemblyName>Microsoft.Azure.Management.ManagedServices</AssemblyName>
+    <PackageTags>Microsoft Azure Managed Services Management;Managed Services;Managed Services Management;</PackageTags>
     <PackageReleaseNotes>
+      <![CDATA[
+      
+      ]]>
     </PackageReleaseNotes>
-	</PropertyGroup>
-	<PropertyGroup>
-		<TargetFrameworks>$(SdkTargetFx)</TargetFrameworks>
```

Comment:
Any reason why this was removed?

Reply:
No I think it is a merge issue. I will add it back.